### ISSUE TITLE
Danish traineddata file doesn't include the "@" character

### ISSUE THREAD

**Issue by Furtifk** [tesseract-ocr/langdata_lstm#29]
### Environment

* **Tesseract Version**: 5.00 Downloaded from: https://github.com/UB-Mannheim/tesseract/wiki
* **Platform**: Windows 10 64bit

### Suggested Fix: Danish traineddata file should include the "@" character

File to run OCR on: ![Screenshot_563](https://user-images.githubusercontent.com/58300000/69877230-e8168700-12c1-11ea-9704-bb0cb44d0cfc.png)

In the case of reproducing I have zip file I can send so you may run a VERY basic test which will display both results comparing eng and dan traineddata results. Please whoever looks into the issue to contact me to receive this.

This is a quite a pressing issue so any response is appreciated.

**Comment by stweil** [comment]
There won't be a fixed `dan.traineddata` soon. I suggest to try `Latin.traineddata` for your case.

**Comment by Furtifk** [comment]
@stweil Thanks for the response.
I will try the latin traineddata although the document I need to be read cannot yield correct results if I use a combination eng + dan traineddata files so I'm not confident this will work. Getting good OCR results for Danish documents seems to be a hassle when not using the Danish dictionary file.

**Comment by stweil** [comment]
It is possible to enhance the existing dan.traineddata with missing characters by additional training, so you could try to fix it yourself. [Here](https://github.com/tesseract-ocr/tesstrain/wiki/GT4HistOCR#finetuning-based-on-scriptfraktur) is a description how this was done for Fraktur. You'll need pairs of line images and text files with a transcription.

**Comment by Furtifk** [comment]
> It is possible to enhance the existing dan.traineddata with missing characters by additional training, so you could try to fix it yourself. [Here](https://github.com/tesseract-ocr/tesstrain/wiki/GT4HistOCR#finetuning-based-on-scriptfraktur) is a description how this was done for Fraktur. You'll need pairs of line images and text files with a transcription.

Thank you for your response. I do not think this is a viable option for me but thanks for your reply and for the information!

**Comment by poizan42** [comment]
It lacks '§' as well which is used in every single legal document in existence...

**Comment by stweil** [comment]
@Furtifk, @poizan42, especially for older Danish texts you could also try one of the models which I trained recently, for example [Fraktur_50000000.502_198857.traineddata](https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/Fraktur_5000000/tessdata_fast/Fraktur_50000000.502_198857.traineddata).

It was trained based on script/Fraktur with lots of historic documents, and according to my experience it works good although I did not add a dictionary. You will get a warning therefore at runtime, but could add a Danish dictionary if needed.

**Comment by Furtifk** [comment]
Has there been any improvements recently with the Danish dictionary?

**Comment by stweil** [comment]
No, and I am afraid there won't be an improvement unless someone works on it.
